keps/sig-storage/596-csi-inline-volumes/README.md

```diff
@@ -213,32 +250,214 @@ Inline volume requests can only participate in mount/unmount volume operations.
+  to be updated to make use of the feature.
+
+  A rollback (or disabling the feature flag) may impact running workloads,
+  as any pod specs taking advantage of ephemeral inline volumes would fail
+  to start when the feature is disabled or the CSIDriver does not support it.
+  It is recommended to delete any pods using this feature before disabling it.
+
+###### What specific metrics should inform a rollback?
+
+  No rollback criteria based on metrics alone. This feature has been enabled by
+  default since 1.16 (beta) and pods that are using ephemeral inline volumes should
+  not start failing as a result of this feature moving to GA.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+  To be documented as part of manual GA testing.
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+  No, this feature does not deprecate any existing functionality.
+
+### Monitoring Requirements
+
+###### How can an operator determine if the feature is in use by workloads?
+
+  Check if any CSIDriver specs have `VolumeLifecycleModes` including an
+  "Ephemeral" `VolumeLifecycleMode` and if any pod specs are using this driver inline.
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [x] Other (treat as last resort)
+  - Details:
+  For pods using an inline CSI driver (as in [Examples](#examples)), review
+  the status of the pod (did it mount and start successfully?).
+  Additionally, the cluster admin may review the kubelet logs for mount/unmount calls.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+- No increased failure rates during mount operations.
+- Mount times should be expected to be less than or equal to the default behavior.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [x] Metrics
+  - Metric name: storage_operation_duration_seconds
+    - [Optional] Aggregation method: filter by `operation_name = volume_mount`
+      - Components exposing the metric: kubelet
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+  No additional metrics needed.
+
+### Dependencies
+
+###### Does this feature depend on any specific services running in the cluster?
+
+  There are no new dependencies introduced by this feature.
+
+### Scalability
+
+###### Will enabling / using this feature result in any new API calls?
+
+  There will be no new API calls. A new API field is added to the CSIDriver
+  API to determine if the driver supports ephemeral inline volumes.
```

Comment:
There is a separate question about new fields in existing APIs - let's remove the second sentence.

Reply:
Done.